guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc

```diff
@@ -8,18 +8,18 @@ include::proc_creating-a-host-in-satellite.adoc[leveloffset=+1]
 
 ifeval::["{build}" == "satellite"]
 [id="editing-the-system-purpose-of-a-host"]
-== Editing the System Purpose of a Host
+== Editing the System Purpose of Hosts
 
-You can edit the system purpose attributes for a Red{nbsp}Hat Enterprise{nbsp}Linux 8 host. System purpose attributes define which subscriptions to attach automatically to this host. For more information about system purpose, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#configuring-system-purpose-standard_configuring-system-settings[Configuring system purpose] in the _Performing a standard RHEL installation_ guide.
+You can edit the system purpose attributes of Red{nbsp}Hat Enterprise{nbsp}Linux 8 hosts. System purpose attributes define which subscriptions to attach automatically to hosts. For more information about system purpose, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#configuring-system-purpose-standard_configuring-system-settings[Configuring system purpose] in the _Performing a standard RHEL installation_ guide.
```

Comment:
I don't think this is limited to RHEL8, afaik you can set syspurpose also for RHEL7